server/blob/blob-cloud/src/test/java/org/apache/james/blob/cloud/CloudBlobsDAOTest.java

```diff
@@ -0,0 +1,91 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.blob.cloud;
+
+import java.net.URI;
+import java.util.Properties;
+import java.util.UUID;
+
+import org.apache.james.blob.api.BlobId;
+import org.apache.james.blob.api.HashBlobId;
+import org.apache.james.blob.api.ObjectStore;
+import org.apache.james.blob.api.ObjectStoreContract;
+import org.jclouds.ContextBuilder;
+import org.jclouds.blobstore.BlobStore;
+import org.jclouds.blobstore.BlobStoreContext;
+import org.jclouds.openstack.keystone.config.KeystoneProperties;
+import org.jclouds.openstack.swift.v1.reference.TempAuthHeaders;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+@ExtendWith(DockerSwiftTempAuthExtension.class)
+public class CloudBlobsDAOTest implements ObjectStoreContract {
+    private static final String TENANT_NAME = "test";
+    private static final String USER_NAME = "tester";
+    private static final Credentials PASSWORD = Credentials.of("testing");
+    private static final Identity IDENTITY = Identity.of(TENANT_NAME + ":" + USER_NAME);
```

Comment:
is this format required ?

Reply:
It is for tempAuth 
for keystone v2 the tenant can be passed in as a header
for keystone v3 the format is needed again. 

Identity will become a more complex structure in the next iteration to support all 3 protocols.